src/plugins/intel_gpu/tests/fusions/gemm_fusion_test.cpp

```diff
@@ -74,6 +74,11 @@ class GemmFusingTest : public ::BaseFusingTest<gemm_test_params> {
     }
 
     layout get_per_channel_layout(gemm_test_params& p) {
+        // WA: refer PR(#15353) message
```

Comment:
Please state it directly here.
"WA: per channel binary post-operation is not supported for Onednn gemm. Use single value for such case.